deployment/ccip/changeset/testhelpers/test_assertions.go

```diff
@@ -333,6 +334,33 @@ func ConfirmCommitWithExpectedSeqNumRange(
 				}
 			}
 		}
+
+		// todo: dedup
+		// -------------------------------------------
+		if len(report.UnblessedMerkleRoots) > 0 {
```

Comment:
This `if` seems unnecessary since the loop just won't run if its length zero or nil, but we gotta clean this up later anyway.